tensorrt_llm/_torch/auto_deploy/shim/ad_executor.py

```
@@ -403,6 +419,25 @@ def create_autodeploy_executor(ad_config: LlmArgs):
     sampler = TorchSampler(sampler_args)
 
+    # Guided (istructured) decoding.
+    guided_decoder = None
+    if (
+        (guided_decoding_backend := ad_config.guided_decoding_backend) is not None
+    ) and dist_mapping.is_last_pp_rank():
+        vocab_size_padded = engine.vocab_size_padded
+        if vocab_size_padded is None:
+            raise RuntimeError(
+                "Could not determine the vocabulary size. Required for guided decoding."
+            )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)